### PR TITLE
Mhp 2117 direct admin new

### PR DESCRIPTION
**Pull request by jesusfreak3**

### CODE REVIEW

**Comment by jesusfreak3** [comment]
Hey ok, this is based off of survey-question-feature branch like the others since the code was going to be mixed a lot in the survey questions

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html [hunk 1]
```diff
@@ -17,17 +17,19 @@
...
+            >
                 <span>
                     {{'ministries.' + tabName + '.header' | t}}
                     <span ng-if="$ctrl[tabName] && $ctrl[tabName] !== 'countHidden'">({{$ctrl[tabName].length}})</span>
                 </span>
                 <loading-spinner ng-if="$ctrl.loadDetails && !$ctrl[tabName]" class="loading" size="16">
                 </loading-spinner>
             </div>
-            <div class="tab dropdown">
+            <div class="tab dropdown" ng-if="$ctrl.directAdminPrivileges">
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html]: So the whole tools menu is hidden now? You could remove the `$ctrl.adminPrivileges` and `$ctrl.directAdminPrivileges` checks nested inside.

Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html]: ah your right, i saw that, planned to do it.. forgot.  Yah the related tickets say all items in the tool list require direct access

Comment (jesusfreak3) [reply app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html]: ok cleaned up

**Pull request merged**